Week02/lesson04/homework/relaxr_starter_code/css/style.css

```diff
@@ -41,44 +42,150 @@ Developer @GA in NYC
   padding: 0 10px 10px 0; /* note the padding around a left floated image */
 }
 
+@font-face {
```

Comment:
Ahhh you got too sophisticated!! Kudos for that, we'll use `@font-face` in the future. `@font-face` is for when we want to host our own fonts (we literally host the files too, like we do the `.css` files or images), whereas here, Google is hosting them for you and you're just using them.

In the HTML you want to add the below to your `<head>`:

`<link href='https://fonts.googleapis.com/css?family=Open+Sans' rel='stylesheet' type='text/css'>`

and then here in the CSS you simply do:

```css
body {
  font-family: 'Open Sans', sans-serif;
}
```